tools/debug-ui/src/BlocksView.tsx

```diff
@@ -0,0 +1,188 @@
+import './BlocksView.scss';
+import { useQuery } from 'react-query';
+import { fetchChainProcessingStatus, BlockProcessingStatus, ChunkProcessingStatus } from './api';
+
+type BlocksViewProps = {
+    addr: string;
+};
+
+function prettyTime(datetime: string) {
+    const time = new Date(Date.parse(datetime));
+    return (
+        time.getUTCHours() +
+        ':' +
+        String(time.getUTCMinutes()).padStart(2, '0') +
+        ':' +
+        String(time.getUTCSeconds()).padStart(2, '0') +
+        '.' +
+        String(time.getUTCMilliseconds()).padStart(3, '0')
```

Comment:
nit. use string.concat()

Reply:
I'm not sure if string.concat() is a common thing to use in js. But you could use string interpolation:
```
`${time.getUTCHours()}:${String(time.getUTCMinutes()).padStart(2, '0')}...
```
it's not amazing either due to the length of the expressions here. So... I'm ok with whatever.

Reply:
Updated to string concat. 

The newline and spaces were added by running prettier